plaso/cli/pinfo_tool.py

```diff
@@ -337,6 +337,7 @@ def _PrintSessionsDetails(self, storage):
     """
     for session_number, session in enumerate(storage.GetSessions()):
       session_identifier = uuid.UUID(hex=session.identifier)
+      session_identifier = str(session_identifier)
```

Comment:
`str()` is not portable since it is bytes on Python 2 and unicode on Python 3 use `'{0!s}'.format()` instead

Reply:
It gives the native string type on each version, but if you want a larger refactor, I'll change the other usages.

Reply:
I've removed the other usages of str() in the file.